src/main/java/com/bunq/sdk/model/generated/Customer.java

```diff
@@ -0,0 +1,161 @@
+package com.bunq.sdk.model.generated;
+
+import com.bunq.sdk.context.ApiContext;
+import com.bunq.sdk.http.ApiClient;
+import com.bunq.sdk.http.BunqResponse;
+import com.bunq.sdk.http.BunqResponseRaw;
+import com.bunq.sdk.model.BunqModel;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Used to view a customer.
```

Comment:
We have `users` Not `customers remember. :grin:

Reply:
We have both users and `Customer`'s; they just mean different things :P

Reply:
generated